### ISSUE TITLE
BUG: blanketFP solid is None if start_angle=stop_angle AND rotation_angle is 360

### ISSUE THREAD

**Issue by RemiTheWarrior** [ukaea/paramak#212]
## How to reproduce
```python
# Doesn't work
test_shape = BlanketFP(
    major_radius=300,
    minor_radius=50,
    triangularity=0.5,
    elongation=2,
    thickness=200,
    stop_angle=360,
    start_angle=0,
)
assert test_shape.solid is not None
# Works
test_shape = BlanketFP(
    major_radius=300,
    minor_radius=50,
    triangularity=0.5,
    elongation=2,
    thickness=200,
    stop_angle=340,
    start_angle=0,
)
assert test_shape.solid is not None
test_shape = BlanketFP(
    major_radius=300,
    minor_radius=50,
    triangularity=0.5,
    elongation=2,
    thickness=200,
    stop_angle=360,
    start_angle=0,
    rotation_angle=180
)
assert test_shape.solid is not None

```

**Comment by RemiTheWarrior** [comment]
@Shimwell @billingsley-john This bug doesn't come up if the connections between points isn't spline but "straight". But I haven't been able to fix anything ...

**Comment by RemiTheWarrior** [comment]
Update: It seems to be due to the fact that spline connections doesn't pass through the exact coordinates of the points making some surfaces to overlap somehow.

**Comment by RemiTheWarrior** [comment]
Fixed in #218
